src/tests/tests/background.js

```diff
@@ -144,11 +141,11 @@
     let done = assert.async(NUM_TESTS);
 
     assert.expect(NUM_TESTS);
+    badger.storage.getNextUpdateForDomain = () => 0;
```

Comment:
Do we need to restore this override?

Reply:
It's done above automatically in the `beforeEach`